experiments/omip_prototype/sixth_degree_omip.jl

```diff
@@ -0,0 +1,150 @@
+using ClimaOcean
+using ClimaSeaIce
+using Oceananigans
+using Oceananigans.Grids
+using Oceananigans.Units
+using Oceananigans.OrthogonalSphericalShellGrids
+using ClimaOcean.OceanSimulations
+using ClimaOcean.ECCO
+using ClimaOcean.JRA55
+using ClimaOcean.DataWrangling
+using ClimaSeaIce.SeaIceThermodynamics: IceWaterThermalEquilibrium
+using Printf
+using Dates
+using CUDA
+
+function synch!(clock1::Clock, clock2)
+    # Synchronize the clocks
+    clock1.time = clock2.time
+    clock1.iteration = clock2.iteration
+    clock1.last_Δt = clock2.last_Δt
+end
+
+synch!(model1, model2) = synch!(model1.clock, model2.clock)
```

Comment:
if you need this please put it in a separate PR and get that merged, because people want to use stuff